Frontend/src/pages/dashboard/components/Farmer/FarmerDashCards.jsx

```
@@ -64,44 +43,151 @@ StatCard.propTypes = {
+        prev30DaysStart.setDate(last30DaysStart.getDate() - 30);
+
+        // Filter orders
+        const last30DaysOrders = orders.filter((order) => new Date(order.date) >= last30DaysStart);
+        const prev30DaysOrders = orders.filter(
+          (order) => new Date(order.date) >= prev30DaysStart && new Date(order.date) < last30DaysStart
+        );
+
+        // Debug prices
+        console.log("Last 30 days orders:", last30DaysOrders);
+        last30DaysOrders.forEach((order, index) => {
+          console.log(`Order ${index} price:`, order.price, "Parsed:", parseFloat(order.price || 0));
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)